packages/components/src/tools-panel/stories/index.tsx

```diff
@@ -112,23 +135,27 @@ export const _default = () => {
 		</PanelWrapperView>
 	);
 };
+Default.args = {
+	label: 'Tools Panel (default example)',
+};
 
-export const WithNonToolsPanelItems = () => {
-	const [ height, setHeight ] = useState();
-	const [ width, setWidth ] = useState();
+export const WithNonToolsPanelItems: ComponentStory< typeof ToolsPanel > = ( {
+	resetAll: resetAllProp,
+	...props
+} ) => {
+	const [ height, setHeight ] = useState< string | undefined >();
+	const [ width, setWidth ] = useState< string | undefined >();
 
-	const resetAll = () => {
+	const resetAll: typeof resetAllProp = ( filters ) => {
 		setHeight( undefined );
 		setWidth( undefined );
+		resetAll( filters );
```

Comment:
```suggestion
		resetAllProp( filters );
```

Small typo but it breaks the reset functionality.

Reply:
https://github.com/WordPress/gutenberg/pull/47944/commits/110b82356098a7b22b5533575b2ff2e423789e4f